pkg/flow/isolate.go

```diff
@@ -58,6 +59,8 @@ const (
 
 	DirektivErrorCodeHeader    = "Direktiv-ErrorCode"
 	DirektivErrorMessageHeader = "Direktiv-ErrorMessage"
+
+	DirektivFunctionContextHeader = "Direktiv-Function-Context"
```

Comment:
I'd like to rename this header. If I understand its purpose correctly, I propose maybe Direktiv-ActionContext. I think that's clearer in the context of the sidecar logic.